projects/canopy/src/lib/button/button.component.scss

```diff
@@ -61,8 +61,9 @@
   }
 
   &--icon-left {
+    flex-direction: row-reverse;
```

Comment:
I suppose the only downside to this approach is that if we were to add a third item inside the button, it might have an unintended affect on the order. E.g. 

```
<button lg-button>
  Submit
  <lg-something>with 10% discount</lg-something>
  <lg-icon name=""></lg-icon>
</button>
```

Or is that a bit of [YAGNI](https://martinfowler.com/bliki/Yagni.html) creeping in? 🙂 

I'm not too worried about this, so would be happy to proceed with this if everyone else is.

Reply:
I think for now it's ok, but you raise a good point. If we ever had a button with an icon at the start and one at the end it would swap them round. However, the current component is only configured to ever have one icon. One to consider if we have a future pattern with multiple icons, but for this I think it's fine.

Reply:
I might be wrong but I think @alexcanning told me buttons will only ever have a single icon.